src/components/Header/Header.jsx

```diff
@@ -23,8 +31,32 @@ const Header = () => {
         <div className="responsive-header">
           <ProfileIcon />
           <Cart />
-          <HamburgerMenu />
+          <HamburgerMenu onClick={toggleMenu} />
         </div>
+        {isOpen && (
+          <div className={`dropdown-menu open slide-in`}>
+            <div className={`menu-overlay open slide-in`} />
```

Comment:
Ne treba ti ista animacija `open slide-in` i za overlay ako vec `dropdown-menu` sluzi za to.

Reply:
Pa cu ti pokazati kasnije sta da radis sa njim

Reply:
Zakomentarisao sam za sada